doc/ref/grpfp.xml

```diff
@@ -107,6 +107,21 @@ A consequence of our convention is that elements of finitely presented
 groups are not printed in a unique way.
 See also <Ref Func="SetReducedMultiplication"/>.
 
+For many <Q>higher level</Q> computations for a finite group,
+such as conjugacy classes or character table,
+it is advisable not to use an FpGroup but an isomorphic group in a better
+internal representation,
+such as a pc group (see <Ref Attr="IsomorphismPcGroup"/>)
+or a permutation group (see <Ref Attr="IsomorphismPermGroup"/>).
+
+In fact,
+calling a function such as <Ref Attr="ConjugacyClasses" Label="attribute"/>
+with an FpGroup that does not store that it is finite
+may result in an error message.
```

Comment:
I think this should explain *why* this is done. Otherwise a user will rightfully wonder "why do they tell me to enter `IsFinite(G)` instead of just doing that for me?"